randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1135,12 +1152,14 @@ Extra - room_id: [32, 35]
               Space Jump
               All of the following:
                   Hi-Jump
-                  Any of the following:
-                      Wall Jump (Beginner)
-                      # https://youtu.be/roSeFDbWr1s
-                      Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
+                  # Climb using vine blocks: https://youtu.be/sQdvfgu3fKA
+                  Wall Jump (Beginner)
+                  # SWJ
+                  Wall Jump (Intermediate) and Can Single Walljump
+                  # Freeze kihunter: https://youtu.be/roSeFDbWr1s
+                  Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
```

Comment:
This block is now completly busted. Requiring HJ and WJ beignner and WJ intermediate and Can freeze enemies?

Also IMO every method that needs to deal with carefully shooting vines should have beginner movement attached.